packages/experimental/Tabs/src/Tabs.types.ts

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { View, ViewProps } from 'react-native';
+import { TextProps } from '@fluentui-react-native/experimental-text';
+import { FocusZoneProps } from '@fluentui-react-native/focus-zone';
+import { FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IShadowTokens } from '@fluentui-react-native/tokens';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
+
+export const tabsName = 'Tabs';
+
+export interface TabsContextData {
+  /*
+   ** The currently selected TabsItem's key
+   */
+  selectedKey: string | null;
+
+  /*
+   ** Index of currently selected key
+   */
+  getTabId?: (key: string, index: number) => string | null;
+
+  /*
+   ** Updates the selected tabsItem and calls the client’s onTabsClick callback
+   */
+  onTabsClick?: (key: string) => void;
+
+  /*
+   ** Updates the selected tabsItem's ref to set as the default tabbable element
+   */
+  updateSelectedTabsItemRef?: (ref: React.RefObject<any>) => void;
+
+  /*
+   ** Array of tabsItem keys in the group
+   */
+  tabsItemKeys?: string[];
+
+  /*
+   ** A Map to for a TabItems corresponding view
+   */
+  views?: Map<string, React.ReactNode[]> | null;
+}
+
+export interface TabsTokens extends IForegroundColorTokens, FontTokens, IBackgroundColorTokens, IShadowTokens {
+  label?: string,
+
+}
+
+
+export interface TabsProps extends  Pick<FocusZoneProps, 'isCircularNavigation'>, Pick<ViewProps, 'accessibilityRole'> {
+  /*
+   ** Descriptive label for the Tabs. This will be displayed as the title of the Tabs to the user
+   */
+  label?: string;
+
+  /*
+   ** The key of the TabsItem that will initially be selected
+   */
+  defaultSelectedKey?: string;
+
+  accessible?: boolean;
+
+  focusable?: boolean;
```

Comment:
I wouldn't think you'd need to add properties like `accessible`,`focusable`, and `accessibilityRole` to a control if you can get them from View Props. Can Tabs just extend ViewProps?